cpp/include/ucxx/request_tag_multi.h

```diff
@@ -218,7 +218,7 @@ class RequestTagMulti : public std::enable_shared_from_this<RequestTagMulti> {
    *
```

Comment:
```suggestion
   *
   * @param[in] status the status of the current request.
```

Reply:
I think the latter makes it more obvious as to what is happening with the request, and it aligns better with the suggestion below too. I've edited the comment and am applying that.